superglue/lib/action_creators/requests.ts

```diff
@@ -206,9 +207,15 @@ export function visit(
           rsp,
           fetchArgs,
         }
+
         const isGet = fetchArgs[1].method === 'GET'
 
-        meta.suggestedAction = 'push'
+        if (action !== undefined) {
+          meta.suggestedAction = action
+        } else {
+          meta.suggestedAction = 'push'
+        }
+
```

Comment:
How do you feel about this change?

```suggestion
meta.suggestedAction = 'push'
```

Then in line 230, after the `suggestionAction` is set, we override it if action was set.

```
if (action) {
  meta.suggestedAction = action
}
```

And since we're allowing for an override, how do you feel about adding a few tests for the override?